CoordgenMinimizer.cpp

```diff
@@ -71,8 +73,25 @@ void CoordgenMinimizer::run()
         setupInteractions();
     }
 
+#ifdef DEBUG_MINIMIZATION_COORDINATES
+    // to seperate energy and DOF minimization
+    energy_list.push_back(-1.f);
+    all_coordinates.push_back({sketcherMinimizerPointF(0,0)});
+#endif
+
     for (int iterations = 0; iterations < m_maxIterations; ++iterations) {
-        scoreInteractions();
+        float energy = scoreInteractions();
+
+#ifdef DEBUG_MINIMIZATION_COORDINATES
+        // store data from this minimization step to be written to a file later
+        energy_list.push_back(energy);
+        std::vector<sketcherMinimizerPointF> these_coordinates;
+        for (auto atom : _atoms) {
+            these_coordinates.push_back(atom->coordinates);
+        }
+        all_coordinates.push_back(these_coordinates);
+#endif
+        energy += 1; // avoid build failure -- will be ok after pushing other PR
```

Comment:
I'll rebase/fix this once https://github.com/schrodinger/coordgenlibs/pull/95 is merged